validator_client/src/initialized_validators.rs

```diff
@@ -254,6 +259,14 @@ impl InitializedValidator {
                     builder
                 };
 
+                let builder = if let Some(path) = client_identity_path {
+                    let identity =
+                        load_pkcs12_identity(path, &client_identity_password.unwrap_or_default())?;
```

Comment:
I'm wondering if we should raise an error here if the password is missing, rather than trying an empty string. A user that forgets to reference a password file will get a `InvalidWeb3SignerClientIdentityCertificate` error, which might not be so helpful.

If it's common practice to use an empty password on this file, I'm happy to go with what you have here :)

Reply:
Agreed. I've also made it an error to provide only the password (previously it was ignored).